provider/client.go

```diff
@@ -40,10 +40,10 @@ func NewClient(host, env, tenant, clientId, clientSecret *string) (*Client, erro
 		env = &environment
 	}
 	hostUrl := "https://api.leanspace.io"
-	switch *env {
-	case "develop", "demo":
-		hostUrl = fmt.Sprintf("https://api.%s.leanspace.io", *env)
-	}
+    switch *env {
```

Comment:
why this change?

Reply:
there is something wrong with my padding, let me check